pages/Threat_Modeling_Process.md

```diff
@@ -33,7 +33,7 @@ permalink: /Threat_Modeling_Process
   - [Subjective Model: DREAD](#subjective-model-dread)
   - [Qualitative Risk Model](#qualitative-risk-model)
 - [Determine Countermeasures and Mitigation](#determine-countermeasures-and-mitigation)
-  - [ASF Threat & Countermeasures Examples](#asf-threat--countermeasures-examples)
+  - [ASF Threat & Countermeasures Examples](https://github.com/OWASP/www-community/blob/master/pages/Threat_Modeling_Process.md#asf-threat--countermeasures-examples)
```

Comment:
This doesn't make sense.
1. Why would a web deployed page link back to GitHub.
2. That anchor link doesn't exist within this page.